mesa/examples/advanced/wolf_sheep/agents.py

```diff
@@ -4,7 +4,7 @@
 class Animal(CellAgent):
     """The base animal class."""
 
-    def __init__(self, model, energy, p_reproduce, energy_from_food, cell):
+    def __init__(self, model, energy=0, p_reproduce=0.2, energy_from_food=1, cell=None):
```

Comment:
What’s the motivation behind these default values?

Reply:
I need default values because you wanted to use keyword arguments on create_agents. These default values have no further meaning then that.